services/prometheus/scrape_configs.go

```diff
@@ -241,43 +241,52 @@ func scrapeConfigForRDSExporter(intervalName string, interval time.Duration, hos
 }
 
 func scrapeConfigsForNodeExporter(s *models.MetricsResolutions, params *scrapeConfigParams) ([]*config.ScrapeConfig, error) {
-	hr, err := scrapeConfigForStandardExporter("hr", s.HR, params, []string{
-		"buddyinfo",
+	var hr, mr, lr *config.ScrapeConfig
+	var err error
+	var hrCollect []string
+
+	if params.node.Distro != "darwin" {
+		mr, err = scrapeConfigForStandardExporter("mr", s.MR, params, []string{
+			"hwmon",
+			"textfile.mr",
+		})
+		if err != nil {
+			return nil, err
+		}
+
+		lr, err = scrapeConfigForStandardExporter("lr", s.LR, params, []string{
+			"bonding",
+			"entropy",
+			"textfile.lr",
+			"uname",
+		})
+		if err != nil {
+			return nil, err
+		}
+
+		hrCollect = append(hrCollect,
+			"buddyinfo",
+			"filefd",
+			"meminfo_numa",
+			"netstat",
+			"processes",
+			"standard.go",
+			"standard.process",
+			"stat",
+			"textfile.hr",
+			"vmstat",
+		)
+	}
+
+	hr, err = scrapeConfigForStandardExporter("hr", s.HR, params, append(hrCollect,
 		"cpu",
```

Comment:
Do only 7 collectors work on MacOS?

Reply:
Yep, only 7 from specified in that file. `textfile.*` can work on macOS by itself, but it's disabled.
https://github.com/percona/pmm-managed/blob/PMM-2.0/services/agents/node.go#L52

Reply:
But where we enable CPU collector on macOS, for example?

Reply:
Ah, it is enabled by default.